src/MergeContext.js

```diff
@@ -298,11 +303,13 @@ class MergeContext {
         const collaborators = await GH.getCollaborators();
         const pushCollaborators = collaborators.filter(c => c.permissions.push === true);
         const requestedReviewers = this._prRequestedReviewers();
+        this._approvalResult = {params: null, delayMs: -1};
 
         for (let collaborator of pushCollaborators) {
             if (requestedReviewers.includes(collaborator.login)) {
                 this._log("requested core reviewer: " + collaborator.login);
-                return null;
+                this._approvalResult.params = {desc: "blocked (see review requests)", state: "error"};
```

Comment:
Like other "waiting" statuses, this should be a `pending`, not `error` status AFAICT (even if GitHub treats review requests as a "red" problem). The `error` status should be used only when there is something wrong with the pull request.